src/universal_transfer_operator/data_providers/database/google/bigquery.py

```diff
@@ -155,6 +240,212 @@ def truncate_table(self, table):
+            transfer_method(
+                source_file=source_file,
+                target_table=target_table,
+                if_exists=if_exists,
+                **kwargs,
+            )
+        else:
+            raise DatabaseCustomError(
+                f"No transfer performed since there is no optimised path "
+                f"for {source_file.location.location_type} to bigquery."
+            )
+
+    def load_gs_file_to_table(
+        self,
+        source_file: File,
+        target_table: Table,
+        if_exists: LoadExistStrategy = "replace",
+        **kwargs,
+    ) -> None:
+        """
+        Transfer data from Google cloud store to bigquery
+
+        :param source_file: Source file that is used as source of data
+        :param target_table: Table that will be created on the bigquery
+        :param if_exists: Overwrite table if exists. Default 'replace'
+        """
+        native_support_kwargs = self.transfer_params.gs_transfer_parameters or {}
+
+        load_job_config = {
+            "sourceUris": [source_file.path],
+            "destinationTable": {
+                "projectId": self.get_project_id(target_table),
+                "datasetId": target_table.metadata.schema,
+                "tableId": target_table.name,
+            },
+            "createDisposition": "CREATE_IF_NEEDED",
+            "writeDisposition": BIGQUERY_WRITE_DISPOSITION[if_exists],
+            "sourceFormat": NATIVE_PATHS_SUPPORTED_FILE_TYPES[source_file.type.name],
+        }
+        if self.is_native_autodetect_schema_available(file=source_file):
+            load_job_config["autodetect"] = True  # type: ignore
+
+        # TODO: Fix this -- it should be load_job_config.update(native_support_kwargs)
+        native_support_kwargs.update(native_support_kwargs)
+
+        # Since bigquery has other options besides used here, we need to expose them to end user.
+        # https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationLoad
+        load_job_config.update(kwargs)
+
+        job_config = {
+            "jobType": "LOAD",
+            "load": load_job_config,
+            "labels": {"target_table": target_table.name},
+        }
+
+        self.hook.insert_job(
+            configuration=job_config,
+        )
+
+    def load_s3_file_to_table(
+        self,
+        source_file: File,
+        target_table: Table,
+        **kwargs,
+    ) -> None:
+        """
+        Load content of multiple files in S3 to output_table in Bigquery by using a datatransfer job
+        1. Enable API on Bigquery
+        2. Enable Data transfer service on Bigquery, which is a chargeable service
+        for more information refer - https://cloud.google.com/bigquery-transfer/docs/enable-transfer-service
+
+        :param source_file: Source file that is used as source of data
+        :param target_table: Table that will be created on the bigquery
+        :param if_exists: Overwrite table if exists. Default 'replace'
```

Comment:
It seems we don't have parm `if_exists` here. Is this intended? Or should we remove it?

Reply:
[Done](https://github.com/astronomer/apache-airflow-provider-transfers/pull/66/commits/49dea8f8e568dc9b61f2dbb54d340c4c42f6ead1)